Tools/parametric_model/src/models/simple_fixedwing_model.py

```diff
@@ -87,8 +87,8 @@ def prepare_force_regression_matrices(self):
         throttle = self.data_df["throttle"].to_numpy()
 
         # vector with cl0, clalpha, cldelta, cd0, cdalpha, cdalpha_sq, ct
-        coeff_vec = np.array([1.0911507971564856, 6.47680410045709, 2.2670575265057487,
-                             0.11373832931995426, 0.7433772294557139, 2.8798589221670072, 85.7567143535296])
+        coeff_vec = np.array([0.07910843279384513, 0.46956829728313765, 0.1643616706716667,
+                             0.00824602887569668, 0.053894849135539054, 0.20878977185710684, 6.217361790630906])
```

Comment:
Why does this need to be hard coded here?

Reply:
no specific reason - the fitted coefficients are not yet available at this point in the pipeline and all this plotting is for debugging only anyways. I'll remove all of it for sure before we merge it (there are corresponding comments in the beginning and end of the two sections)

Reply:
The framework already provides predictions with estimated parameters right? There is also a separate app where it can do prediction without estimation

Reply:
It does, but I think I broke the plotting at some point for the new model and didn't fix it yet. I didn't know about the separate app though. I will get it done until next week at the latest.